docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,291 @@
+    $ sudo systemctl start docker.service
+
+## Verify Docker installation
+
+    $ sudo docker run hello-world
+
+*Optional:* You may also want to create a `docker` group and add your local user to the group.  This means you can drop the `sudo` in the `docker ...` commands that follow.
+
+    $ sudo groupadd docker
+    $ sudo usermod -aG docker `whoami`
+    $ sudo systemctl restart docker.service
+
+Then log out (`exit`) and log back in to pick up your new group association.  Verify your user has access to Docker without sudo
+
+    $ docker ps
+
+## Set & verify fully qualified domain name
+
+These instructions assume each host can reach other hosts using their fully qualified domain names (FQDN).  To check the FQDN on a host use
+
+    $ hostname -f
+
+Then attempt to ping that name from other servers.
+
+Also important are that Calico and Mesos have the same view of the (non-fully-qualified) hostname.  In particular, the value returned by
+
+    $ hostname
+
+must be unique for each node in your cluster.  Both Calico and Mesos use this value to identify the host.
+
+## Configure your firewall
+
+You will either need to configure the firewalls on each node in your cluster (recommended) to allow access to the cluster services or disable it completely.  Included in this section is configuration examples for `firewalld`.  If you use a different firewall, check your documentation for how to open the listed ports.
+
+Master node(s) require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| zookeeper    | 2181/tcp          |
+| mesos-master | 5050/tcp          |
+| etcd         | 2379/tcp 4001/tcp |
+| marathon     | 8080/tcp          |
+
+Example `firewalld` config
+
+    $ sudo firewall-cmd --zone=public --add-port=2181/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=5050/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=2379/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=4001/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=8080/tcp --permanent
+    $ sudo systemctl restart firewalld
+
+Agent (compute) nodes require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| BIRD (BGP)   | 179/tcp           |
+| mesos-agent  | 5051/tcp          |
+
+Example `firewalld` config
+
+    $ sudo firewall-cmd --zone=public --add-port=179/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=5051/tcp --permanent
+    $ sudo systemctl restart firewalld
+
+# Install Mesos
+
+## Build the packages
+
+The Mesos RPM packages can be built from the Mesosphere [net-modules repo](https://github.com/mesosphere/net-modules).
+
+    $ git clone git@github.com:mesosphere/net-modules.git
+    $ cd net-modules
+    $ make builder-rpm
+    $ ls packages/rpms/RPMS/x86_64
+
+The files you see in the `packages/rpms/RPMS/x86_64` directory are all the packages you need to install Mesos. Transfer these files to each host in your cluster.
```

Comment:
...you need to install Mesos, while additionally including the compiled net-modules library. Transfer these...